src/Matrix.jl

```diff
@@ -3558,6 +3558,8 @@ If `side` is `:right` and `flag` is true, $B$ is the right inverse of $A$
 i.e. $AB$ is the $n\times n$ unit matrix. If `side` is `:left` and `flag` is
 true, $B$ is the left inverse of $A$ i.e. $BA$ is the $m\times m$ unit matrix.
 If `flag` is false, no right or left inverse exists.
+
+To get the space of all inverses, combine with [`nullspace`](@ref).
```

Comment:
This is perhaps a bit too cryptic in this condensed form? How about something
like this:
```suggestion
To get the space of all inverses, note that if $B$ and $C$ are
both right inverses, then $A(B-C)=0$, and similar for left inverses.
Hence from one inverse one can find all by making suitable use of
[`nullspace`](@ref).
```

Reply:
Yeah, that's better. Thanks!